django-stubs/contrib/auth/hashers.pyi

```diff
@@ -15,30 +15,55 @@ def identify_hasher(encoded: str) -> BasePasswordHasher: ...
 def mask_hash(hash: str, show: int = ..., char: str = ...) -> str: ...
 
 class BasePasswordHasher:
-    algorithm: str
-    library: str | tuple[str, str]
+    algorithm: None
```

Comment:
```suggestion
    algorithm: str | None
```

Reply:
I applied this change, thanks for your suggestion :)